tests/cggi/syntax.mlir

```diff
@@ -0,0 +1,21 @@
+// RUN: heir-opt -verify-diagnostics %s
```

Comment:
`-verify-diagnostics` does not do anything unless you have specific IR comments that assert messages are written to stderr. https://mlir.llvm.org/getting_started/TestingGuide/#diagnostic-verification-tests

Ditto for lwe/types.mlir

Reply:
Done